### ISSUE TITLE
Issues after testing plugin with WP 5.7 RC release

### ISSUE THREAD

**Issue by pooja-muchandikar** [rtCamp/rtMedia#1726]
- [ ] 1. Following error is shown on navigating to media related pages
[08-Mar-2021 15:35:58 UTC] PHP Notice:  WP_Scripts::localize was called <strong>incorrectly</strong>. The <code>$l10n</code> parameter must be an array. To pass arbitrary data to scripts, use the <code>wp_add_inline_script()</code> function instead. Please see <a href="https://wordpress.org/support/article/debugging-in-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 5.7.0.) in /var/www/htdocs/wp-includes/functions.php on line 5313

- [ ] 2.

**Comment by pooja-muchandikar** [comment]
### Update: 

```
Test Environment
rtMedia: fix/issue-1726 branch
BuddyPress: v7.2.1
WordPress: 5.7
Theme: 2021
```

## Issues

- [ ] With the recent fixes in fix/issue-1726 branch on visiting [media](https://puja-rtmedia.dev4.rt.gw/members/pooja/media/) page following console error is shown. 

![image](https://user-images.githubusercontent.com/41000648/112585574-12c61080-8e20-11eb-96d0-027be3d1670b.png)

---

- [ ] The WP_scripts warning is not fixed on activity page. Also there is major issue the `Upload` button from media page is not working. 
Please check the visual: https://drive.google.com/file/d/1YrBLbo8cm18O5GsdsLcSUTWEFF2a-PKC/view

**Comment by pooja-muchandikar** [comment]
@OpuRockey 

The above issue is fixed but changes not reflecting. Still the same issue is shown. Can you once please confirm?

`[13-Sep-2021 10:18:33 UTC] PHP Notice:  WP_Scripts::localize was called <strong>incorrectly</strong>. The <code>$l10n</code> parameter must be an array. To pass arbitrary data to scripts, use the <code>wp_add_inline_script()</code> function instead. Please see <a href="https://wordpress.org/support/article/debugging-in-wordpress/">Debugging in WordPress</a> for more information. (This message was added in version 5.7.0.) in /var/www/htdocs/wp-includes/functions.php on line 5663`

**Comment by OpuRockey** [comment]
@pooja-muchandikar , I didn't find that. Can you check with that person who worked previously for this issue?

**Comment by pooja-muchandikar** [comment]
Cross-checked the issue again and seems like it was cache error. Now not getting any warnings.

**Comment by pooja-muchandikar** [comment]
Cross-checked this issue with `develop` branch and the fix looks fine.

**Comment by pooja-muchandikar** [comment]
Cross-checked the issue with `master` branch and works as expected.

**Comment by pooja-muchandikar** [comment]
Issue fixed and released in version `4.6.7`
